pwd-service/src/main/resources/templates/contact.twig

```diff
@@ -6,27 +6,31 @@
             <div class="col-sm-8 col-sm-offset-2">
                 <h1>Kontakt</h1>
 
-                <form role="form">
+                <form role="form" action="/kontakt" method="post">
                     <div class="form-group">
-                        <input type="text" class="form-control" id="name" name="name" placeholder="Imię i Nazwisko"
+                        <input type="text" class="form-control" id="name" name="name" placeholder="Imię i Nazwisko" value="{{ name }}"
```

Comment:
Wydawało mi się, że w ten sposób wysyłają się wartości na submicie, ale rozumiem, że dzieje się to po id, prawda?